League Sandbox Auto Setup/Util/Utility.cs

```diff
@@ -18,5 +19,25 @@ public static void CreateShortcut(string shortcutName, string shortcutPath, stri
             shortcut.TargetPath = targetFileLocation;                 // The path of the file that will launch when the shortcut is run
             shortcut.Save();                                    // Save the shortcut
         }
+
+        public static void WaitForFile(Action action, int timeoutMs = 300000)
+        {
+            var time = Stopwatch.StartNew();
+            while (time.ElapsedMilliseconds < timeoutMs) // Default timeout is 5 mins
+            {
+                try
+                {
+                    action();
+                    return;
+                }
+                catch (IOException e)
+                {
+                    // access error
+                    if (e.HResult != -2147024864)
```

Comment:
Is there a nicer way to write the magic number `-2147024864`?

Reply:
Tried to get it through errlook.exe in Visual Studio tools, all solutions which I found in stackoverflow just based on this value, we could maybe set it as const "accessError"?